tests/test-general.el

```diff
@@ -897,50 +897,36 @@ Return t if successful or a cons corresponding to the failed key and def."
     (expect (general-test-keys 'normal general-temp-map
               "a" #'a)))
   (it "should correctly extract definitions from general definer arglists"
-    (expect (plist-get (use-package-normalize/:general
-                        nil
-                        nil
-                        '((general-def "key" #'command)))
-                       :commands)
-            :to-equal '(command))
-    (expect (plist-get (use-package-normalize/:general
```

Comment:
Add this test back.

Reply:
@noctuid 

Since `use-package-normalize/:general` just returns the argument as is I renamed this series of tests to use `use-package-autoloads/:general`. After the rename this test case was identicle to the first test below in `should extract symbols depending on general-use-package-emit-autoloads` so I removed it. It might be worth combining these 3 tests into the next test series since `should correctly extract definitions from general definer arglists` doesn't really describe what it does now.

Reply:
Done.